tests/helpers/load_state_tests.py

```diff
@@ -69,7 +69,10 @@ def run_blockchain_st_test(test_case: Dict, load: Load) -> None:
                 break
 
         if block_exception:
-            with pytest.raises(InvalidBlock):
+            # TODO: Once all the specific exception types are thrown,
+            #       only `pytest.raises` the correct exception type instead of
+            #       all of them.
+            with pytest.raises((InvalidBlock, InvalidTransaction)):
                 add_block_to_chain(chain, json_block, load, mock_pow)
```

Comment:
Wondering if we can make it more specific on our test consumption side as well. Treat `InvalidBlock` and `InvalidTransaction` separately based on `block_exception`. Not urgent right now but some thing to consider going forward.

Reply:
That's what I was thinking, and intended to convey with the above TODO. We can't do it yet because I haven't converted all the transaction exceptions yet (just one specific case.)